### ISSUE TITLE
TypeError: Cannot read property 'source' of undefined.

### ISSUE THREAD

**Issue by sanonz** [icelam/html-inline-script-webpack-plugin#1]
It does not work when `inject: false` is used.

```js
new HtmlWebpackPlugin({
  filename: './index.html'
}),
new HtmlWebpackPlugin({
  inject: false,
  filename: './supports.html'
})
```

**Comment by icelam** [comment]
Hi @sanonz, thank you for reporting the issue. I have just release a fix in version 1.0.1. Let me know if the fix is working or not ☺️

**Comment by sanonz** [comment]
thx, It's working very well.
